src/BuildScriptGeneratorCli/Program.cs

```diff
@@ -11,13 +11,13 @@
 namespace Microsoft.Oryx.BuildScriptGeneratorCli
 {
     [Command("oryx", Description = "Generates and runs build scripts for multiple languages.")]
-    [Subcommand(BuildCommand.Name,           typeof(BuildCommand))]
-    [Subcommand(PlatformsCommand.Name,       typeof(PlatformsCommand))]
-    [Subcommand(BuildScriptCommand.Name,     typeof(BuildScriptCommand))]
-    [Subcommand(RunScriptCommand.Name,       typeof(RunScriptCommand))]
-    [Subcommand(InstallRunTimeCommand.Name,  typeof(InstallRunTimeCommand))]
-    [Subcommand(BuildpackDetectCommand.Name, typeof(BuildpackDetectCommand))]
-    [Subcommand(BuildpackBuildCommand.Name,  typeof(BuildpackBuildCommand))]
+    [Subcommand(typeof(BuildCommand))]
+    [Subcommand(typeof(PlatformsCommand))]
+    [Subcommand(typeof(BuildScriptCommand))]
+    [Subcommand(typeof(RunScriptCommand))]
+    [Subcommand(typeof(InstallRunTimeCommand))]
+    [Subcommand(typeof(BuildpackDetectCommand))]
+    [Subcommand(typeof(BuildpackBuildCommand))]
```

Comment:
what is this change for? we don't need descriptions anymore?

Reply:
the previous constructor we used is now deprecated